components/form/TaskForm.tsx

```diff
@@ -1,19 +1,20 @@
 import { useUpdateTasks } from '@/lib/hooks/useTasks';
 import { Project, Task } from '@/lib/types/apiTypes';
 import { FormBase } from '@/lib/types/forms';
-import { FormControl, FormLabel, Input, FormErrorMessage, Button, Box } from '@chakra-ui/react';
+import { FormControl, FormLabel, Input, FormErrorMessage, Button, Box, Flex, Checkbox } from '@chakra-ui/react';
 import React, { useState } from 'react';
 import ErrorAlert from '../common/ErrorAlert';
 
 type TaskFields = Partial<Task> & { project: Project };
```

Comment:
It would probably be a good idea to edit this type to
```
type TaskFields = Partial<Task> & { project: Project, billable: boolean };
```

Reply:
This type represents the minimum information information that the initial state of a form must have. If we add `billable` field, it means that the form must always have `billable` set to something when it's initialized.